src/main/scala/mesosphere/marathon/state/PathId.scala

```diff
@@ -121,6 +121,8 @@ sealed trait PathId extends Ordered[PathId] with plugin.PathId with Product {
 
 case class AbsolutePathId(path: Seq[String]) extends PathId {
   override val absolute: Boolean = true
+
+  @deprecated("Calling this method is a no-op")
```

Comment:
Do we still need this method?

Reply:
I think we should be able to get rid of it!